### ISSUE TITLE
Move Land Ownership data to Layers section

### ISSUE THREAD

**Issue by lin-d-hop** [DigitalCommons/land-explorer-front-end#136]
## Description
We treat LandOwnership data as a layer, but currently it has its own section of the UI (with much of the contents not functional)
Let's treat it like a real layer and add it to the layers section.

A new section: 
Land Ownership (alongside Land Data)
- Land Registry 

## Acceptance Criteria
- Move the Land Registry ownership data layer to the layers section
- A new section (alongside Land Data and Resilient Green Spaces) called `Land Ownership`
- One option beneath that called `Land Registry`
- When a property is selected the information about it displays in the Info panel. On clicking a property the Info menu panel is automatically opened and the information displayed.

**Comment by lin-d-hop** [comment]
The intention was to put in the Info menu panel (added to acceptance criteria). Would you be up for having a crack at that for feedback or is it too much and designs would be helpful?

**Comment by King-Mob** [comment]
I'll have a go with the info!

**Comment by King-Mob** [comment]
@lin-d-hop i've put a version on staging with first 3 acceptance criteria fulfilled. 
Question about what moves into the info box from land ownership box:
highlight multiple toggle? I was thinking about maybe just having it always set to highlight multiple.
clear all button? 
I'll have a further go tomorrow

**Comment by lin-d-hop** [comment]
I think defaulting to multiple toggle would be great. This feels like a superfluous UI option to me.

A few notes:
- Once Land Ownership layer is selected let's have it stay 'on' until turned 'off'
- When a property is selected, let's auto-open the info box with the info
- The info box might want sections now if it is doing drawn objects and properties?
   -> Just like the polys have 'Polygon1' with the chevron to view deatils, can we have 'Property [INSPIRE ID] and a chevron to view all the property information?
   -> Can we use the deeds symbol here just as we use the poly symbol beside the poly?
- Maybe not a clear all box, but a 'Clear/Remove' button in the dropdown for the specific property

Lots of notes, maybe too complex for a comment?

**Comment by King-Mob** [comment]
This all makes sense

**Comment by King-Mob** [comment]
@lin-d-hop up on staging. There aren't named or collapsible sections, but drawn objects and properties are grouped together in the box. so if you have lots it'll go
Marker Marker Marker Marker Polygon Polygon Polygon Property Property Property
regardless of what order you opened them in.

**Comment by lin-d-hop** [comment]
A very cool first step.

Here is where we start to get into the curly stuff.... let's keep chatting as we explore this:)

It currently doesn't play well with drawing tools. Steps to reproduce:
1. Turn on the Property ownership layer
2. Select some properties
-> It would be great if the Info panel auto opened at this point so that people can see the info for the properties they selected.
(Side note - zoom out and the properties stay selected which is really cool!)
4. Go to the drawing tools
5. Select the poly tool
6. Click about. Properties keep selecting even though the drawing tool is selected.
7. There is some odd behaviour if I click enough. Occassional line when I click the line tool for example.

Use case: I select the property and then want to draw on the map using the property boundary as a guide. I can't do this.

I appreciate we are getting into curly territory. Like I say above let's keep discussing. I am happy that we might need to make decisions on trade offs as we go through this implementation. Let's focus on keeping good design patterns in the code as this is implemented so if there are code complexity trade offs let's talk them through. What I want to avoid is curly code that makes this work and everything in the future becomes impossibly complex :)

**Comment by King-Mob** [comment]
What I am getting from this is:
- properties don't highlight on click when the drawing tools are open
- highlighting a property opens the info panel, if not already open 
I don't think those two are especially curly

**Comment by lin-d-hop** [comment]
I would add to that:
- All drawing tools work correctly when property ownership layer is on

Glad to hear this seems straightforward to you :)

**Comment by lin-d-hop** [comment]
@King-Mob 
One more slight tweak please -
The land ownership layer appears under User Group layers. Please can we group 'universal' layers together at the top and 'user specific' layers together below that?

**Comment by lin-d-hop** [comment]
@King-Mob The PRs are huge improvements! Thank you!

I've noticed a weird thing in testing. On this property when I selected it, 3 properties selected at the same time. It might be specific to this property. 

https://user-images.githubusercontent.com/8669848/206939830-3a604033-acf9-4777-b7ad-7a810fd6990e.mp4

Would it be better to open a separate bug for this so we don't hold up the other PRs merging?

Will continue testing and report anything else I notice.
